### PR TITLE
Created User.java

### PR DESCRIPTION
**Pull request by mohamedabdelsalam**

### CODE REVIEW

File: src/main/java/csed2025/User.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package csed2025;
+
```
Comment (mohamedabdelsalam) [inline src/main/java/csed2025/User.java:2]: Please write java doc. 

rename the name of the class to `CalUser`

**Review by mohamedabdelsalam — COMMENTED** [review]
please follow my reviews

File: src/main/java/csed2025/User.java (hunk 1 above)
Comment (mohamedabdelsalam) [reply src/main/java/csed2025/User.java:2]: - agree with u for java doc. 

- CalUser isn't a perfect name

**Review by abderrahman504 — APPROVED** [review]

**Review by abderrahman504 — APPROVED** [review]
Agreed

**Pull request merged**